src/site/stages/build/process-cms-exports/schemas/transformed/node-outreach_asset.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+module.exports = {
+  $id: 'node-outreach_asset',
+  type: 'object',
+  properties: {
+    // Standard `node` key-value pairs.
+    changed: {
+      type: 'number',
+    },
+    // Unsure how this key-value pair is derived from the CMS export data.
+    // entityID: {
+    //   type: 'string',
+    // },
+    status: {
+      type: 'boolean',
+    },
+    title: {
+      type: 'string',
+    },
+    // `outreach_asset`-specific key-value pairs.
+    fieldBenefits: {
+      $ref: 'nullable-string',
```

Comment:
I removed this in my PR after I made it. :sweat_smile: 

Using [multiple types](https://cswr.github.io/JsonSchema/spec/multiple_types/) is the right way to do it:
```js
{ type: ['string', 'null'] }
```